domain/src/main/java/uk/gov/hmcts/ccd/definition/store/domain/validation/event/EventEntityValidationContext.java

```diff
@@ -12,10 +15,13 @@
 
     private final String caseReference;
 
+    private final List<String> caseRoles;
+
     public EventEntityValidationContext(CaseTypeEntity parentCaseType) {
         this.caseName = parentCaseType.getName();
         this.caseReference = parentCaseType.getReference();
         this.parentSecurityClassification = parentCaseType.getSecurityClassification();
+        this.caseRoles = parentCaseType.getCaseRoles().stream().map(caseRole -> caseRole.getReference()).collect(Collectors.toList());
```

Comment:
I think you need to add the global case roles to this list. Otherwise, global case roles will be rejected.

Reply:
DONE